inscriptions/ISic000647.xml

```diff
@@ -200,28 +205,56 @@
+                        reason="lost"><expan><abbr>Χρ</abbr><ex>ιστ</ex><abbr>ῷ</abbr></expan> ζήσ</supplied>ασα (for this formula, see e.g. IMC Catania 192 = <ref target="http://sicily.classics.ox.ac.uk/inscription/ISic003290">ISic003290</ref>, IG 14.535 = <ref target="http://sicily.classics.ox.ac.uk/inscription/ISic001354">ISic001354</ref> and Korhonen 2004: 263).</note></app>
+                    <app loc="lines 5-6"><note>Libertini: <expan><abbr>ὅρκι</abbr><ex>ον</ex></expan> | <supplied reason="lost">αὐτο</supplied>κράτορα; Ferrua: ὀρκί-|<supplied reason="lost">ζω (sic) <expan><abbr>θ</abbr><ex>εὸν</ex></expan> παντο</supplied>κράτορα; Manganaro: ὁρκί-|<supplied reason="lost">ζω τὸν Παντο</supplied>κρατορα; Korhonen: ὀρκί-|<supplied reason="lost">ζω (sic) τὸν παντο</supplied>κράτορα</note></app>
+                    <app loc="line 7"><note>Libertini, Manganaro: <supplied reason="lost">τὸν μέλ</supplied>λοντα; Ferrua, Korhonen: <supplied reason="lost">τὸν ἔτι μέλ</supplied>λοντα</note></app>
+                    <app loc="line 8"><note>Ferrua: <supplied reason="lost">τὴν σορὸν μη</supplied>δέν'; Manganaro: <supplied reason="lost">κληρονόμων μη</supplied>δέν'. Korhonen reads a vertical stroke before ΕΝ (Η, Ι, Ν, Ω)</note></app>
+                    <app loc="lines 9-10"><note>Ferrua: <supplied reason="lost">ἐγεννήθη τῇ <expan><abbr>πρ</abbr><ex>ὸ</ex></expan></supplied><gap reason="lost" quantity="1" unit="character"/> <expan><abbr>νων</abbr><ex>ῶν</ex></expan>, κατ<supplied reason="lost">ε</supplied>|<supplied reason="lost">τέθη</supplied><gap reason="lost" extent="unknown" 
+                        unit="character"/>; Manganaro: <supplied reason="lost">ἢ σκυλήσαι</supplied><gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost">ἐκεί</supplied>νων κατ<supplied reason="lost">ά</supplied>|<supplied reason="lost">θεσιν</supplied> (or κατ<supplied reason="lost">ά|γειον μνημίον</supplied>)<gap reason="lost" extent="unknown" unit="character"/>; Korhonen: <gap 
+                        reason="lost" extent="unknown" unit="character"/> <supplied reason="lost" cert="low">ἐκεί</supplied>νων κατ<supplied reason="lost">ά</supplied>|<supplied reason="lost" cert="low">θεσιν</supplied><gap reason="lost" extent="unknown" unit="character"/></note></app>
+                </listApp>
+            </div>
+            <div type="translation" xml:lang="en" resp="#martaf">
+                <p><gap reason="lost"/>all the world <gap reason="lost"/><supplied reason="subaudible">of so and so</supplied> and of Basilis. They lived perfectly and blamelessly with me fourteen years <gap reason="lost"/>lived twenty-eight years. I beseech in the name of the Almighty <gap reason="lost"/>for all eternity <gap reason="lost"/>let no one open <supplied reason="lost">the tomb</supplied><gap reason="lost"/></p>
```

Comment:
simply: [---] all the world [--- of (name)] and of Basilis.
and so on - no mark-up, just written out in plain text.